website/content/v1.8/kubernetes-guides/configuration/replicated-local-storage-with-openebs.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Replicated Local Storage"
+description: "Using local storage with OpenEBS"
+aliases:
+  - ../../guides/storage
+  - ./replicated-local-storage-with-openebs-jiva
+---
+
+If you want to use replicated storage leveraging disk space from a local disk with Talos Linux installed, OpenEBS is a great option.
+
+Since OpenEBS is a replicated storage, it's recommended to have at least three nodes where sufficient local disk space is available.
+The documentation will follow installing OpenEBS via the offical Helm chart.
+Since Talos is different from standard Operating Systems, the OpenEBS components need a little tweaking after the Helm installation.
+Refer to the OpenEBS [documentation](https://openebs.io/docs/quickstart-guide/installation) if you need further customization.
+
+> NB: Also note that the Talos nodes need to be upgraded with `--preserve` set while running OpenEBS, otherwise you risk losing data.
+> Even though it's possible to recover data from other replicas if the node is wiped during an upgrade, this can require extra operational knowledge to recover, so it's highly recommended to use `--preserve` to avoid data loss.
+
+## Preparing the nodes
+
+Create a machine config patch with the contents below and save as `patch.yaml`
+
+```yaml
+machine:
+  sysctls:
+    vm.nr_hugepages: "1024"
+  nodeLabels:
+    openebs.io/engine: mayastor
+  kubelet:
+    extraMounts:
+      - destination: /var/local/openebs
```

Comment:
This path is incorrect based on the standard helm chart...should be `/var/openebs/local`. Change must be made below as well.

Reply:
can we reopen or should I make another PR?

Reply:
This PR is merged, so please open another PR

Reply:
Done https://github.com/siderolabs/talos/pull/9155